packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -108,6 +108,6 @@ export const messages = {
   force_project_create_text: 'SFDX: Create Project',
   force_project_create_open_dialog_create_label: 'Create Project',
   force_apex_trigger_create_text: 'SFDX: Create Apex Trigger',
-  no_refresh_if_currently_active:
-    'Refresh is already active.  Cancel the existing task if there is a need to restart'
+  force_sobjects_no_refresh_if_already_active_error_text:
```

Comment:
Move this under line 107 so it's next to the other sobject text. Feel free to merge after that.